src/BuildingRegistry/Building/BuildingUnit.cs

```diff
@@ -432,6 +432,40 @@ public void DetachAddressBecauseAddressWasRemoved(AddressPersistentLocalId addre
                 newAddressPersistentLocalId));
         }
 
+        public void Demolish()
```

Comment:
Maybe we should also create a partial class here, it's a long file :) ? 
`BuildingUnit_Grb` ?